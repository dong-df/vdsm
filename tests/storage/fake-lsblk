#!/bin/sh
set -e

# Fake the output
cat $FAKE_STDOUT
