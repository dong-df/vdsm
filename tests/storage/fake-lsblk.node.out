{
  "blockdevices": [
    {
      "name": "/dev/mapper/0QEMU_QEMU_HARDDISK_host4-data2",
      "type": "part",
      "children": [
        {
          "name": "/dev/mapper/0QEMU_QEMU_HARDDISK_host4-data",
          "type": "mpath",
          "children": [
            {
              "name": "/dev/sda",
              "type": "disk"
            }
          ]
        }
      ]
    }
  ]
}